slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
   - Supporting projects: CLI, Universal, Nativescript
 - A complete and opinionated solution
 - You don't need to be a Javascript expert
-- Can be integrated with Redux
\ No newline at end of file
+- Can be integrated with Redux
+
+
+---
+
+## Getting Set Up
+
+1. Install NVM (Node Version Manager) -- [nvm.sh](http://nvm.sh)
+1. Install the latest LTS version of node -- `nvm install --lts`
+1. Install Angular CLI -- `npm install --global @angular/cli`
```

Comment:
I'll add some extra notes for Windows users. It seems that NVM for windows is not maintained